.dev/gather_benchmark_train_results.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import glob
+import os.path as osp
+
+import mmcv
+from gather_models import get_final_results
+from mmcv import Config
```

Comment:
Does `get_final_results` work for `xxx.log.json` of the latest version?
```
{"mode": "train", "epoch": 430, "iter": 159800, "lr": 0.00012, "memory": 18550, "data_time": 0.03098, "decode.final.loss_seg": 0.1267, "decode.final.acc_seg": 89.37481, "decode.4x.loss_seg": 0.05068, "decode.4x.acc_seg": 89.37481, "decode.cascade_1.loss_seg": 0.05473, "decode.cascade_1.acc_seg": 89.02916, "decode.cascade_2.loss_seg": 0.06245, "decode.cascade_2.acc_seg": 88.45142, "loss": 0.29455, "time": 0.40958}
{"mode": "train", "epoch": 430, "iter": 159850, "lr": 0.00012, "memory": 18550, "data_time": 0.01244, "decode.final.loss_seg": 0.12426, "decode.final.acc_seg": 90.30909, "decode.4x.loss_seg": 0.0497, "decode.4x.acc_seg": 90.30909, "decode.cascade_1.loss_seg": 0.05396, "decode.cascade_1.acc_seg": 89.95941, "decode.cascade_2.loss_seg": 0.06163, "decode.cascade_2.acc_seg": 89.38377, "loss": 0.28955, "time": 0.45076}
{"mode": "train", "epoch": 430, "iter": 159900, "lr": 0.00011, "memory": 18550, "data_time": 0.01909, "decode.final.loss_seg": 0.12876, "decode.final.acc_seg": 89.90795, "decode.4x.loss_seg": 0.0515, "decode.4x.acc_seg": 89.90795, "decode.cascade_1.loss_seg": 0.05563, "decode.cascade_1.acc_seg": 89.55699, "decode.cascade_2.loss_seg": 0.06348, "decode.cascade_2.acc_seg": 88.96176, "loss": 0.29938, "time": 0.20294}
{"mode": "train", "epoch": 430, "iter": 159950, "lr": 0.00011, "memory": 18550, "data_time": 0.04161, "decode.final.loss_seg": 0.12413, "decode.final.acc_seg": 90.28908, "decode.4x.loss_seg": 0.04965, "decode.4x.acc_seg": 90.28908, "decode.cascade_1.loss_seg": 0.0539, "decode.cascade_1.acc_seg": 89.94204, "decode.cascade_2.loss_seg": 0.06162, "decode.cascade_2.acc_seg": 89.37222, "loss": 0.2893, "time": 0.48495}
{"mode": "train", "epoch": 431, "iter": 160000, "lr": 0.0001, "memory": 18550, "data_time": 0.02879, "decode.final.loss_seg": 0.12311, "decode.final.acc_seg": 90.08211, "decode.4x.loss_seg": 0.04924, "decode.4x.acc_seg": 90.08211, "decode.cascade_1.loss_seg": 0.05334, "decode.cascade_1.acc_seg": 89.74749, "decode.cascade_2.loss_seg": 0.06098, "decode.cascade_2.acc_seg": 89.16612, "loss": 0.28667, "time": 0.34189}
{"mode": "val", "epoch": 431, "iter": 125, "lr": 0.0001, "aAcc": 0.9527, "mIoU": 0.727, "mAcc": 0.8069, "IoU.road": 0.9799, "IoU.sidewalk": 0.8352, "IoU.building": 0.9073, "IoU.wall": 0.4933, "IoU.fence": 0.5505, "IoU.pole": 0.5568, "IoU.traffic light": 0.6097, "IoU.traffic sign": 0.7214, "IoU.vegetation": 0.9115, "IoU.terrain": 0.5975, "IoU.sky": 0.9395, "IoU.person": 0.7605, "IoU.rider": 0.5405, "IoU.car": 0.9367, "IoU.truck": 0.759, "IoU.bus": 0.816, "IoU.train": 0.6353, "IoU.motorcycle": 0.5443, "IoU.bicycle": 0.7187, "Acc.road": 0.9881, "Acc.sidewalk": 0.9236, "Acc.building": 0.9546, "Acc.wall": 0.5448, "Acc.fence": 0.6566, "Acc.pole": 0.6921, "Acc.traffic light": 0.7166, "Acc.traffic sign": 0.802, "Acc.vegetation": 0.9651, "Acc.terrain": 0.6985, "Acc.sky": 0.9758, "Acc.person": 0.8721, "Acc.rider": 0.7069, "Acc.car": 0.9719, "Acc.truck": 0.8161, "Acc.bus": 0.8869, "Acc.train": 0.6681, "Acc.motorcycle": 0.6356, "Acc.bicycle": 0.8555}
```

Reply:
Fixed.
You can refer to this [line](https://github.com/sennnnn/mmsegmentation/blob/27866dcbc0df938cec35a8aa311e1440ad34b498/.dev/gather_models.py#L63)